tests/forte/processors/stave_processor_test.py

```diff
@@ -0,0 +1,95 @@
+# Copyright 2019 The Forte Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Unit tests for stave processor.
+"""
+
+import os
+import unittest
+import threading
+import requests
+
+from typing import Any, Dict, Iterator, Optional, Type, Set, List
+from forte.data.data_pack import DataPack
+from forte.pipeline import Pipeline
+from forte.data.readers import OntonotesReader
+from forte.processors.base import PackProcessor, FixedSizeBatchProcessor
+from forte.processors.base.batch_processor import Predictor
+from ft.onto.base_ontology import Token, Sentence, EntityMention, RelationLink
+from forte.processors.stave import StaveProcessor
+from forte.processors.third_party import NLTKWordTokenizer, \
+    NLTKPOSTagger, NLTKSentenceSegmenter
+
+
+class TestStaveProcessor(unittest.TestCase):
+
+    def setUp(self):
+
+        os.environ["FRONTEND_BUILD_PATH"] = "stave/build/"
+        os.environ["ONTOLOGY_PATH"] = "forte/ontology_specs/base_ontology.json"
+        os.environ["DJANGO_BACKEND_PATH"] = "stave/simple-backend/"
+
+        self._port: int = 8880
+        self._project_name: str = "serialization_pipeline_test"
+
+        self.pl = Pipeline[DataPack]()
+        self.pl.set_reader(OntonotesReader())
+        self.pl.add(NLTKSentenceSegmenter())
```

Comment:
Actually let's start not using these 3rd party processors in our tests if not necessary, because we are going to refactor and move them out.